lib/asciidoctor/converter/semantic-html5.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+module Asciidoctor
+# A built-in {Converter} implementation that generates HTML 5 output
+# that maximizes the use of semantic constructs.
+class Converter::SemanticHtml5Converter < Converter::Base
+  register_for 'semantic-html5', 'sem-html5'
```

Comment:
Let's make a separate issue for the backend name. I'm fine if we have a working name in the interim, but I don't think we'll land on semantic-html5 when it's all said and done.

Reply:
@mojavelinux Not sure how I should phrase it. Maybe we should create an issue to discuss the transition?
- find a backend name for this new converter
- discuss when and how this new converter will become the default one
- find a backend name for the current HTML5 converter if we decide to rename it

Reply:
Yes please. I'm fine with using an interim name as long as we're actively discussing what we want it to be (even if we decide to stick with this one). We may, in fact, need two issues. One to decide on the name(s) (which has certain implications in how intrinsic attributes are computed) and one for how we will manage the transition.

Reply:
I've created two follow-up issues.